saleor/graphql/order/types.py

```diff
@@ -523,7 +525,22 @@ class OrderLine(ModelObjectType):
     thumbnail = graphene.Field(
         Image,
         description="The main thumbnail for the ordered product.",
-        size=graphene.Argument(graphene.Int, description="Size of thumbnail."),
+        size=graphene.Argument(
```

Comment:
Definition of the `thumbnail` field seems the same in all cases, I was wondering if it wouldn't be nice to define a `ThumbnailField` that extends the `graphene.Field` and predefines the `size` and `format` arguments with these descriptions? Not sure how easy/hard it is to add, but might be worth considering.